src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
```

Comment:
You create Lottery 3 times inside the loop. We can have only 1 Lottery object and get 3 balls from it. 
We do not need three different markets to buy three milk bottles) 
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/ddc791be-6620-457a-aa6e-ad58e076b7ad)